integration_tests/macros/e2e_tests/test_config_levels.sql

```diff
@@ -0,0 +1,53 @@
+{% test config_levels(model, expected_config, timestamp_column, time_bucket, where_expression, anomaly_sensitivity, anomaly_direction, days_back, backfill_days, seasonality, min_training_set_size) %}
```

Comment:
The test is executed 3 times:
- 2 tests on the same model, one with test config and the other with model config. The validation is that the model level configs override the project level, and the test level configs override the model level config. 
- 1 test on a model with no specific config, to see that the project defaults are picked up.